packages/php/compat-checker/src/Checks/WPCompatibility.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * WordPress Compatibility Check.
+ *
+ * @package Automattic/WooCommerce/Grow/Tools
+ */
+
+namespace Automattic\WooCommerce\Grow\Tools\CompatChecker\v0_0_1\Checks;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The WooCommerce Compatibility Check class.
+ */
+class WPCompatibility extends CompatCheck {
```

Comment:
❓ I can imagine it would almost never be a problem IRL, but should there be a check for the WP requires at least header? Or do we let the `WC requires at least` set the WP version requirement too?

Reply:
I had added this check earlier but while testing I noticed that WordPress core has the `WP requires at least` check internally and deactivates the incompatible plugins. So I've removed our redundant implementation. Thank you for checking this.

Reply:
👍🏻  That makes sense. It's a pretty basic check, so I suspected there must be some reason to leave it out.